autoreject/autoreject.py

```diff
@@ -892,7 +892,6 @@ def fit(self, epochs):
                     local_reject.mean_ = _slicemean(
                         epochs_interp[train].get_data()[:, self.picks],
                         good_epochs_idx, axis=0)
-                    X = epochs[test].get_data()[:, self.picks]
```

Comment:
Really ? I are still scoring on the test samples?

Reply:
I am not sure what you mean

Reply:
now X is `X = epochs.get_data()[:, self.picks]` while before it was `X = epochs[test].get_data()[:, self.picks]` so it seems score is now called on full epochs not just the test epochs

Reply:
forget it. I just saw the X[test] below...

Reply:
I updated it after seeing your comment :) But just wanted to say there is no bug on master